rest_sensors.cpp

```diff
@@ -1574,8 +1666,8 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                             {
                                 data = QByteArray("\x01", 1);
                             }
-                            data.append((qint8)(setting[1].toUInt()));
-                            data.append((qint8)(setting[2].toUInt()));
+                            data.append(static_cast<qint8>(setting[1].toUInt()));
+                            data.append(static_cast<qint8>(setting[2].toUInt()));
```

Comment:
The list isn't checked for to contain 3 values, if the size is < 3 it will crash deCONZ.

```
if (settings.size() == 3 && setting[0].type() == QVariant::Bool && setting[1].type() == QVariant::Double && setting[2].type() == QVariant::Double)
{
}
```

Reply:
done